File: FixTest.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;

public class FixTest {
    public static void main(String[] args) throws IOException {
        String path = "e:/projects/E-learning/src/test/java/com/edtech/platform/enrollment/EnrollmentTest.java";
        String content = new String(Files.readAllBytes(Paths.get(path)));
        content = content.replace("import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;", "import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;\nimport static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.user;");
        content = content.replace(".header(HttpHeaders.AUTHORIZATION, \"Bearer \" + token)", ".with(user(studentDetails))");
        Files.write(Paths.get(path), content.getBytes());
    }
}

File: fix_test.py
import os
path = 'src/test/java/com/edtech/platform/dashboard/DashboardIntegrationTest.java'
with open(path, 'r') as f: c = f.read()

c = c.replace('import com.edtech.platform.course.entity.PublishStatus;', 'import com.edtech.platform.course.enums.PublishStatus;')
c = c.replace('import com.edtech.platform.enrollment.entity.EnrollmentStatus;', 'import com.edtech.platform.enrollment.enums.EnrollmentStatus;')

with open(path, 'w') as f: f.write(c)
